.github/workflows/test.yml

```
@@ -121,7 +121,7 @@ jobs:
     - name: Clone MNE-Python
       run: git clone --depth 1 --single-branch --branch main https://github.com/mne-tools/mne-python.git
```

Depth 1 breaks version number
```suggestion
      run: git clone --single-branch --branch main https://github.com/mne-tools/mne-python.git
```